### PR TITLE
Added troubleshooting tips

### PR DESCRIPTION
**Pull request by raghav4**
Added troubleshooting tips to deal with the installation error caused by `bcrypt` dependency.

### CODE REVIEW

**Comment by yochannah** [comment]
thanks for the PR @raghav4! 

@arunans23 has offered to take a look at this and may have some comments :)

**Comment by raghav4** [comment]
> thanks for the PR @raghav4!
> 
> @arunans23 has offered to take a look at this and may have some comments :)

Thank you @yochannah @arunans23

File: views/addInstance.ejs [hunk 1]
```diff
@@ -278,6 +278,8 @@
 							$("#newUrl").val(instance.url);
 							$("#newDesc").val(instance.description);
 							$("#maintainerOrgName").val(instance.maintainerOrgName);
+							$("#maintainerEmail").val(instance.maintainerEmail);
+							$("#maintainerGithubUrl").val(instance.maintainerGithubUrl);
 							$("#maintainerUrl").val(instance.maintainerUrl);
```
Comment (arunans23) [inline views/addInstance.ejs]: Don't we have to add here as well https://github.com/intermine/intermine-registry/blob/dev/views/addInstance.ejs#L178 ?

File: app.js [hunk 2]
```diff
@@ -112,4 +112,4 @@ app.use(function(err, req, res, next) {
   res.render('error');
 });
 
-module.exports = app;
+module.exports = app;
```
[the remark this answers is not in the document]
Comment (raghav4) [reply app.js]: Okay @arunans23  👍 😄

File: views/addInstance.ejs (hunk 1 above)
Comment (raghav4) [reply views/addInstance.ejs]: No, I think because in the issue the only update was mentioned. If you want me to add, I can add it. Should I add it then?

Comment (arunans23) [reply views/addInstance.ejs]: I think we should. @danielabutano any thoughts?

Comment (raghav4) [reply views/addInstance.ejs]: @arunans23 , yes it is another issue which occurs while a warning is generated whenever any existed instance is added. Fixing it too!

Comment (raghav4) [reply views/addInstance.ejs]: Done @arunans23 , you can review the PR now.

Comment (danielabutano) [reply views/addInstance.ejs]: Good spot @arunans23 ! thanks for the contribution @raghav4

**Pull request merged**